<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>361</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTextEdit" name="textEdit_eq"/>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_result"/>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="1">
         <widget class="QPushButton" name="Button_5">
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QPushButton" name="Button_point">
          <property name="text">
           <string>.</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="Button_minus">
          <property name="text">
           <string>-</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="Button_7">
          <property name="text">
           <string>7</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="Button_4">
          <property name="text">
           <string>4</string>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QPushButton" name="Button_divide">
          <property name="text">
           <string>/</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="Button_3">
          <property name="text">
           <string>3</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="Button_AC">
          <property name="text">
           <string>AC</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="Button_6">
          <property name="text">
           <string>6</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QPushButton" name="Button_0">
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="Button_2">
          <property name="text">
           <string>2</string>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QPushButton" name="Button_plus">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="Button_8">
          <property name="text">
           <string>8</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="Button_1">
          <property name="text">
           <string>1</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QPushButton" name="Button_9">
          <property name="text">
           <string>9</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QPushButton" name="Button_square">
          <property name="text">
           <string>*</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="Button_C">
          <property name="text">
           <string>C</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2" colspan="2">
         <widget class="QPushButton" name="Button_equal">
          <property name="text">
           <string>=</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
